hs_core/templates/search/indexes/hs_core/baseresource_text.txt

```diff
@@ -25,9 +25,15 @@
 {% for res in object.metadata.time_series_results %}{{ res.sample_medium|discoverable }} {{ res.units_name }} {{ res.units_type }}  {% endfor %}
 {% for f in object.logical_files %} 
 {{ f.get_discovery_content_type }} {{ f.metadata.cellinformation.name|discoverable }}  
-{% for v in f.metadata.variables.all %}{{ v.name|discoverable }} {% endfor %} 
+{% for v in f.metadata.variables.all %}{{ v.name|discoverable }} {{ v.code|discoverable }} {% endfor %} 
```

Comment:
What happens if the logical file metadata does not have variables?  Does it error out and not create the rest of the document or does it just continue to the next block?

Reply:
This is specifically crafted so it doesn't error out. The one way it can error out is if you use an iterator as an atom or an atom as an iterator. This is the reason for the if statement below, because the time series resource uses the same label differently and errors out.